GameData/RealismOverhaul/RO_SuggestedMods/SXT/RO_SXT_Engines.cfg

```diff
@@ -167,6 +167,105 @@
 	}
 	engineType = J2
 }
+// HG-3
++PART[SXTLT80]:BEFORE[RealismOverhaul]
+{
+	@name=HG-3
+	%RSSROConfig = True
+	{
+	}
```

Comment:
Module was deleted in https://github.com/KSP-RO/RealismOverhaul/pull/1609/commits/90d1cb68280406f80f2e82ddbe25ad6e73fac924 but you forgot the brackets